src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java

```diff
@@ -154,7 +154,7 @@ public JavaResolveResult advancedResolve(boolean incompleteCode) {
     //
 
     // For the moment (while debugging the resolver) let's do this without caching.
-    boolean skipCaching = false;
+    boolean skipCaching = true;
```

Comment:
Yes, please do.  It should be false unless we are debugging, though I know it gets checked in set to true from time to time.